pkg/cbfs/empty.go

```diff
@@ -18,6 +18,7 @@ func NewEmptyRecord(f *File) (ReadWriter, error) {
 	r := &EmptyRecord{File: *f}
 	Debug("Got header %v", r.String())
 	r.Type = TypeDeleted2
+	r.Name = ""
```

Comment:
no need to do this r.Name, that's how it will be initialized. This is not C, thank goodness :-)